conf/messages.cy

```diff
@@ -425,7 +425,7 @@ groupAlreadyEnrolled.body.p2.link = <a href="{0}" class="govuk-link">ychwanegu
 agentCannotRegister.title = Ni allwch ddefnyddio’r gwasanaeth hwn i gofrestru ar gyfer yr Ardoll Troseddau Economaidd
 agentCannotRegister.heading = Ni allwch ddefnyddio’r gwasanaeth hwn i gofrestru ar gyfer yr Ardoll Troseddau Economaidd
 agentCannotRegister.p1 = Ni all asiantau cwblhau cofrestriad ar gyfer yr Ardoll Troseddau Economaidd (ECL) ar ran cleient.
-agentCannotRegister.p2 = Mae’n rhaid i’ch cleient gofrestru drwy ddefnyddio’i Dynodydd Defnyddiwr (ID) ar gyfer Porth y Llywodraeth ei hun.
+agentCannotRegister.p2 = Rhaid i'ch cleient gofrestru gan ddefnyddio ei ID defnyddiwr a'i gyfrinair ei hun.
```

Comment:
I think the single quotes there needs to be duplicated. You'll have to check it live and see.